src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +117,28 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sorts the List based on name (first priority).
+     * If names are the same, sort by contact numbers.
+     */
+    public void sort() {
+        internalList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                String firstName = o1.getName().fullName;
+                String secondName = o2.getName().fullName;
+                String firstPhone = o1.getPhone().value;
+                String secondPhone = o2.getPhone().value;
+                int notEqual = firstName.compareTo(secondName);
+                if (notEqual == 0) {
+                    return firstPhone.compareTo(secondPhone);
+                } else {
+                    return notEqual;
+                }
```

Comment:
maybe instead of using an if else statement, you could check if the names are the same at the top?

e.g.
if (firstName.compareTo(secondName) == 0) {
    return firstPhone.compareTo(secondPhone);
}
return firstName.compareTo(secondName);

Reply:
Thank you for your comment! I assigned firstName.compareTo(secondName) == 0 to a variable to so as to not repeat the code below. Save computation time, but will look into making my code neater!